src/PointwiseFunctions/AnalyticData/GrMhd/CylindricalBlastWave.hpp

```diff
@@ -130,12 +128,12 @@ class CylindricalBlastWave : public MarkAsAnalyticData {
       default;
   ~CylindricalBlastWave() = default;
 
-  CylindricalBlastWave(
-      InnerRadius::type inner_radius, OuterRadius::type outer_radius,
-      InnerDensity::type inner_density, OuterDensity::type outer_density,
-      InnerPressure::type inner_pressure, OuterPressure::type outer_pressure,
-      MagneticField::type magnetic_field, AdiabaticIndex::type adiabatic_index,
-      const OptionContext& context = {});
+  CylindricalBlastWave(double inner_radius, double outer_radius,
+                       double inner_density, double outer_density,
+                       double inner_pressure, double outer_pressure,
+                       std::array<double, 3> magnetic_field,
```

Comment:
I believe we usually take non-fundamental types by const-ref? (I suppose there is an argument to be made that a std:array<double, 3> is a small type and should be ok?). For consistency I would suggest changing to a const-ref here and elsewhere throughout the PR.